### PR TITLE
[Entry] SASS - New term page for sass concept

### PR DESCRIPTION
**Pull request by lloan**
### Description

New term page for SASS concept, to go under the CSS term page. 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/sass/sass.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  font-size: 24px;
+  color: #333;
+}
+```
+
+## **Key Features**
+
+### **Variables** 
+
+SASS allows you to define reusable variables, making it easy to store and reuse values throughout your stylesheets.
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/css/concepts/sass/sass.md:48]: we avoid the use of first and second person pronouns ( I, you, we) in docs. Please review and revise the other instances in this entry as above.

File: content/css/concepts/sass/sass.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+.disabled-button {
+  @extend .button;
+  opacity: 0.5;
+  cursor: not-allowed;
+}
+```
+
+### **Operators**
+
+Operators in SASS are symbols that perform mathematical operations on values. They include arithmetic operators (+, -, *, /), comparison operators (==, !=, >, <), and logical operators (and, or, not). They enable dynamic calculations and conditional logic within SASS stylesheets.
```
Comment (caupolicandiaz) [inline content/css/concepts/sass/sass.md:173]:
```suggestion
Operators in SASS are symbols that perform mathematical operations on values. They include arithmetic operators ( `+, -, *, / `), comparison operators (`==, !=, >, <`), and logical operators (`and, or, not`). They enable dynamic calculations and conditional logic within SASS stylesheets.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@lloan I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@lloan Looks good. I made a couple of minor tweaks, but overall this is in good shape. I'm passing this on for a second review.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@lloan Only a few minor changes from me.

**Review by KTom101 — APPROVED** [review]

**Comment by KTom101** [comment]
@lloan Thank you for your contribution 🎉 Link is live here: https://www.codecademy.com/resources/docs/css/sass

Screenshot 2023-07-27 at 11 12 37 PM

**Pull request merged**